### DISCUSSION TITLE
E_CANNOT_GET_METADATA: Unable to retrieve the \"exists\" for file at location \"xxx.pdf\""

### DISCUSSION THREAD

**Discussion by tsulatsitamim** [adonisjs/core#3606] [Help]
HI, i try to check if file exist in s3, buat I got this error. how to resolve this? thank you

![2022-03-04_22-10](https://user-images.githubusercontent.com/5946791/156788363-3ae1bddd-32ba-4339-bf1b-a3c9f2b8fd7a.png)
this is my aws user policy

**Comment by ghani-othman** [comment]
@thetutlage answer this please..if file existed it will return true but it return catch while file doesnot exist..it should return false right?
